### PR TITLE
MotoRos v1.9.5

### PR DESCRIPTION
**Pull request by EricMarcil**
- Fixed message reporting PFL being enabled after E-Stop.
- Added check on startup to identify if PFL is enabled, and remove PFL related checks if disabled.
- Added MotionStop if controller is unable to process incremental motion to prevent inaccurate incomplete motion.
- Improved handling of waking robot from energy saving mode (Eco Mode) in relation with improvement  made on controller software YAS4.13 (YRC1000) or later and YBS2.43 (YRC1000micro) or later.

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
Tested on our YRC1000 (`YAS4.24`) -- with an FSU, but no PFL.

Sequence described by @ted-miller in #410 is no longer problematic (ie: traj mode can be enabled).

No others problems encountered, but I'll have to rely on someone else to test the PFL side to this.

**Comment by gavanderhoorn** [comment]
For future PRs: would it be possible to use separate commits instead of a single one? That would make reviews a little easier.

**Comment by EricMarcil** [comment]
@gavanderhoorn Thanks for the testing this.  I tested on my HC10 with PFL.  I've also test on YRC1000 and DX200 without PFL.
Sorry for the single commit. It started as a single issue and I was doing testing, I kept finding other small issues to fix.  I'll try to split them in the future.

**Review by gavanderhoorn — APPROVED** [review]

File: motoman_driver/MotoPlus/SimpleMessage.h [hunk 1]
```diff
@@ -159,7 +159,8 @@ typedef enum
 	ROS_RESULT_NOT_READY_NOT_STARTED,
 	ROS_RESULT_NOT_READY_WAITING_ROS,
 	ROS_RESULT_NOT_READY_SKILLSEND,
-	ROS_RESULT_NOT_READY_PFL_ACTIVE
+	ROS_RESULT_NOT_READY_PFL_ACTIVE,
+	ROS_RESULT_NOT_READY_INC_MOVE_ERROR
```
Comment (gavanderhoorn) [inline motoman_driver/MotoPlus/SimpleMessage.h:163]: We need to update the `NotReadyCode` enum:

https://github.com/ros-industrial/motoman/blob/9ade90acfe5835ca38fddcf6ac9cf589a452f449/motoman_driver/include/motoman_driver/simple_message/motoman_motion_reply.h#L100-L116

Comment (gavanderhoorn) [reply motoman_driver/MotoPlus/SimpleMessage.h:163]: We'll do that in a separate PR.

**Comment by gavanderhoorn** [comment]
@EricMarcil: will you update the wiki page (ie: the links to the `.out` files and the versions)?

**Comment by gavanderhoorn** [comment]
Thanks for the update @EricMarcil :+1:

**Comment by EricMarcil** [comment]
@gavanderhoorn: Yes, we'll update wiki page.  I've requested edit access.  If I still don't have access on Monday, I'll ask Ted (He usually the one that updates the wiki...  But I should probably get access to do it for my changes.)

**Comment by gavanderhoorn** [comment]
This is the page that needs to be updated: [motoman_driver/Tutorials/indigo/InstallServer](http://wiki.ros.org/motoman_driver/Tutorials/indigo/InstallServer). The last update changed this: [diff](http://wiki.ros.org/action/info/motoman_driver/Tutorials/indigo/InstallServer?action=diff&rev2=63&rev1=62).

That includes the version nrs, links to the binaries and the `md5sum`s.

**Comment by EricMarcil** [comment]
Wiki was updated

**Comment by gavanderhoorn** [comment]
thank you

**Comment by EricMarcil** [comment]
I just realized that I forgot to change the version number in the MotoPlus code (in MotoROS.h) but I don't think it actually displays anywhere.

**Comment by gavanderhoorn** [comment]
Hm. It should actually. IIRC, @ted-miller added a bit of code which registers MotoROS with the system to show versioning info on the TP *somewhere*.

I'll try to find it.

**Comment by gavanderhoorn** [comment]
Here:

https://github.com/ros-industrial/motoman/blob/0786691078e75e5b3a5b17543ee6013fdf40d568/motoman_driver/MotoPlus/Controller.c#L77-L93

**Comment by ted-miller** [comment]
Yeah, the version number displays on the pendant and is also exported to SYSTEM.SYS and PANELBOX.LOG when saving a robot backup. It's helpful for troubleshooting to know the exact version the user is using.

**Comment by gavanderhoorn** [comment]
@EricMarcil: if you could submit a fixup-PR with the version nr updated and new binaries, I'll review it quickly.

**Comment by EricMarcil** [comment]
@gavanderhoorn I just pushed PR #413 to fix it.

File: motoman_driver/MotoPlus/SimpleMessage.h (hunk 1 above)
Comment (gavanderhoorn) [reply motoman_driver/MotoPlus/SimpleMessage.h:163]: See #438.

**Pull request merged**